recipes/hdmf/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "hdmf" %}
+{% set version = "0.post0.dev1081" %}
+{% set sha256 = "8b0264a51ef1aea5fb22d450a7fa6cbc74bac97421e9667d1b7caee0ab74ce57" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
This should be as here: https://conda-forge.org/docs/meta.html#use-pip

Reply:
done

Reply:
For my own education, how is `PYTHON` supposed to be set? I assumed Jinja would pull it from the environment, but it wasn't able to find it.